### PR TITLE
Fixed ordered list prefix (MD029) - Group 1

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes all MD029 errors in the following guides:

- REST
- Test
- UI Components

The MD)29 rule requires all ordered list prefixes to be consistent. We've chosen "one" as our prefix. The scope of #5248 also permits—but does not require—resolving errors related to the following rules:

- MD030: Spaces after list markers

This is one of three PRs related to #5248.

### CODE REVIEW

File: guides/v2.2/test/js/test_js-unit.md [hunk 1]
```diff
@@ -153,26 +158,32 @@ JsTestDriver
...
 **JsTestDriver output:**
 
-    $ php <magento2_root_dir>dev/tests/js/run_js_tests.php
-    java -jar C:\Users\mchiocca\lib\JsTestDriver.jar --config C:\git\magento2\dev\tests\js/jsTestDriver.conf --port 9876 --browser "C:\Program Files (x86)\Mozilla Firefox\firefox.exe" --tests all --testOutput C:\git\magento2\dev\tests\js/test-output
-    setting runnermode QUIET
-    ....................................
-    Total 36 tests (Passed: 36; Fails: 0; Errors: 0) (138.00 ms)
-      Firefox 15.0 Windows: Run 36 tests (Passed: 36; Fails: 0; Errors 0) (138.00 ms)
+```bash
+$ php <magento2_root_dir>dev/tests/js/run_js_tests.php
```
Comment (keharper) [inline guides/v2.2/test/js/test_js-unit.md]: Wrap this line in a `bash` block and remove the $.

Wrap line 173 in a separate `bash` block.

Wrap lines 174 -177 in a `terminal` block.

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_template_literals.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ contributor_name: SwiftOtter Studios
 contributor_link: https://swiftotter.com/
 ---
 
-# Template Literals in Magento
+## Template Literals in Magento
```
Comment (keharper) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_template_literals.md]: There's no reason to have this heading.

File: guides/v2.2/test/js/test_js-unit.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.2/test/js/test_js-unit.md]: Done

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_template_literals.md (hunk 2 above)
Comment (jeff-matthews) [reply guides/v2.2/ui_comp_guide/concepts/ui_comp_template_literals.md]: Agreed. Removed

**Pull request merged**